cmd/tekton-mcp-server/main.go

```diff
@@ -55,18 +63,26 @@ func main() {
 	resources.Add(ctx, s)
 
 	slog.Info("Starting the server.")
-	// Start the stdio server
-	stdioServer := server.NewStdioServer(s)
-	// Start listening for messages
+
 	errC := make(chan error, 1)
-	go func() {
-		in, out := io.Reader(os.Stdin), io.Writer(os.Stdout)
 
-		errC <- stdioServer.Listen(ctx, in, out)
-	}()
+	if transport == "sse" {
+		sseServer := server.NewSSEServer(
+			s,
+			server.WithSSEContextFunc(func(_ context.Context, r *http.Request) context.Context { return ctx }),
+		)
+		go func() {
+			errC <- sseServer.Start(sseAddr)
+		}()
 
-	// Output tekton-mcp string
-	_, _ = fmt.Fprintf(os.Stderr, "Tekton MCP Server running on stdio\n")
+		slog.Info(fmt.Sprintf("Tekton MCP Server is listening at %s", sseAddr))
+	} else {
+		go func() {
+			errC <- server.ServeStdio(s, server.WithStdioContextFunc(func(_ context.Context) context.Context { return ctx }))
```

Comment:
Swapped to use https://pkg.go.dev/github.com/mark3labs/mcp-go@v0.23.1/server#ServeStdio as it seemed simpler and did the same thing, I can swap back if desired.

Reply:
That makes sense, thank you for explaining! Reverted back to the original code